src/components/Lecturers/Lecturers.elements.tsx

```diff
@@ -0,0 +1,81 @@
+import styled from "styled-components"
+import { colors } from '../../styled/variables'
+
+export const CardsContainer = styled.div`
+    display: flex;
+    justify-content: center;
+    margin: 0 auto 30px auto;
+    flex-wrap: wrap;
+`
+
+export const Card = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 285px;
+    height: 317px;
+    border-radius: 15px;
+    box-shadow: 0px 12px 25px 0 rgba(0, 0, 0, 0.1);
+    background-color: ${colors.white};
+    margin-bottom: 20px;
+    position: relative;
+    margin-left: 10px;
+    margin-right: 10px;
+
+   :hover .hoverLecturer { 
```

Comment:
Mám za to, že obvykle se používá spíš `${LecturerImgColor}` - vkládá se do textu přímo `styled-element`...Popsané je to přímo tady https://styled-components.com/docs/advanced#referring-to-other-components